PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -288,7 +336,7 @@ def fetch_response(apiCall):
 
 
 def check_neighbors(step):
```

Comment:
Maybe add a docstring here?
Not sure what this function does

Reply:
Ok,
I am fine with the implementation of this function for now :-)
Just remember that if you will ever need to change more than 2 neighbors change this to be dynamic and don't write another function.